### PR TITLE
Rename project according to new defined name

### PR DESCRIPTION
**Pull request by fedeagripa**
New name defined, so need to change almost everything to then push the gem properly

### CODE REVIEW

**Comment by santiagovidal** [comment]
Shouldn't it be RS instead of RT? 🤔

**Comment by fedeagripa** [comment]
RT was previously approved in the gem channel, but if everyone want to change it I can

File: lib/rt_rails_base.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+BASE_PATH = 'rt-rails-base'.freeze
```
Comment (grilix) [inline lib/rt_rails_base.rb]: I'd avoid this constant here.

**Comment by fedeagripa** [comment]
@grilix all set

**Pull request merged**